src/nxt_conf_validation.c

```diff
@@ -210,6 +210,11 @@ static nxt_int_t nxt_conf_vldt_isolation(nxt_conf_validation_t *vldt,
 static nxt_int_t nxt_conf_vldt_clone_namespaces(nxt_conf_validation_t *vldt,
     nxt_conf_value_t *value, void *data);
 
+#if (NXT_HAVE_CGROUP)
```

Comment:
I'm not convinced by the practice of providing prototypes conditionally.  I would like to get rid of it, so that we can get rid of preprocessor conditionals at call site too.

Reply:
Gone.

Reply:
I've kept the one in src/nxt_isolation.c ias we only want to call nxt_isolation_set_cgroup() when cgroups are supported, just like all the function calls in nxt_isolation_set() which are called conditionally.

Reply:
Okay.

Reply:
Yeah, I guess you need it beacuse if not, then you get errors due to unused static functions.  The solution to that (in my mind) would be to transform the preprocessor conditionals into normal C conditionals, but that's not possible as of now.  So yes, this makes sense to me.

Reply:
Indeed.